### PR TITLE
Add Chapter Parameterization-specific Output

### PR DESCRIPTION
**Pull request by JulieSchramm**
html and pdf builds are successful

### CODE REVIEW

**Comment by climbfuji** [comment]
> html and pdf builds are successful

Can I look at the HTML and PDF? Way easier to check than the raw rst files. Thanks ...

**Comment by JulieSchramm** [comment]
Assuming you have the Sphinx, etc. software on your desktop:

git clone https://github.com/JulieSchramm/ccpp-doc.git
cd ccpp-doc/CCPPtechnical
make html
make latexpdf
open build/html/index.html (this will open in a browser window)
open build/latex/CCPPtechnical.pdf (this will open in preview)

**Comment by climbfuji** [comment]
> I reviewed the HTML and all looks in order. I participated in writing the content, so my review is from an "insider" point of view.

For this PR and future PRs, you can easily attach files to a comment on GitHub - so please attach either the PDF or HTML (whatever GitHub allows) so that the reviewer(s) don't have to run the sphinx commands. I don't even know whether I set it up in the past. Thank you ...

**Comment by JulieSchramm** [comment]
[CCPPtechnical.pdf](https://github.com/NCAR/ccpp-doc/files/5001760/CCPPtechnical.pdf)

File: CCPPtechnical/source/ParamSpecificOutput.rst [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+over the number of hours specified by the user in variable ``fhzero`` in the ``&gfs_physics_nml`` portion of the
+namelist file ``input.nml``. Variable ``fhzero`` must be an integer (it cannot be zero). 
+
+Outputting Tendencies
+---------------------
+
+UFS
+^^^
+
+When ``ldiag3d`` and/or ``qdiag3d`` are set to true, the tendencies described in
```
Comment (climbfuji) [inline CCPPtechnical/source/ParamSpecificOutput.rst:164]: This and/or is misleading - beforehand you are saying that if qdiag3d is true, then ldiag3d must be true, too.

File: CCPPtechnical/source/ParamSpecificOutput.rst [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+
+Further documentation of the ``diag_table`` file can be found in the UFS Weather Model User’s Guide
+`here <https://ufs-weather-model.readthedocs.io/en/latest/InputsOutputs.html#diag-table-file>`_.
+
+After running, the requested arrays will be present in the output files.  
+
+SCM
+^^^
+
+The default behavior of the SCM is to output instantaneous values of all variables in
```
Comment (climbfuji) [inline CCPPtechnical/source/ParamSpecificOutput.rst:187]: Is this still correct after Grant's update to the tracer handling? I assume yes, but want to make sure. @grantfirl

File: CCPPtechnical/source/ParamSpecificOutput.rst [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+
+Physics-side changes
+^^^^^^^^^^^^^^^^^^^^
+
+In order to output auxiliary arrays, developers need to change at least the following
+two files within the physics (see also example in :numref:`Section %s <CodeModExample>`):
+
+* A CCPP entrypoint scheme
+   * Add array(s) and its/their dimension(s) to the list of subroutine arguments
+   * Declare and allocate array(s)
```
Comment (climbfuji) [inline CCPPtechnical/source/ParamSpecificOutput.rst:279]: No, the user does not allocate arrays - the allocation takes place in `GFS_typedefs.F90` (automatically). You declare the arrays with the appropriate `intent` and dimensions.

File: CCPPtechnical/source/ParamSpecificOutput.rst [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+   +  long_name = auxiliary 2d arrays to output (for debugging)
+   +  units = none
+   +  dimensions = (horizontal_dimension,number_of_3d_auxiliary_arrays)
+   +  type = real
+   +  kind = kind_phys
+   +[aux3d]
+   +  standard_name = auxiliary_3d_arrays
+   +  long_name = auxiliary 3d arrays to output (for debugging)
+   +  units = none
+   +  dimensions = (horizontal_dimension,vertical_dimension,number_of_3d_auxiliary_arrays)
```
Comment (climbfuji) [inline CCPPtechnical/source/ParamSpecificOutput.rst:419]: I wonder if we should mention that the vertical dimension of these auxiliary arrays is hardcoded to the standard `vertical_dimension`, which may be incompatible with other vertical dimensions used (i.e. the vertical dimension for radiation).

Comment (ligiabernardet) [reply CCPPtechnical/source/ParamSpecificOutput.rst:419]: The introductory paragraph says " all diagnostics are output on physics model levels. An extension to enable output on radiation levels may be considered in future implementations." Is this sufficient?

Comment (climbfuji) [reply CCPPtechnical/source/ParamSpecificOutput.rst:419]: Sorry, I must have missed that part. Thanks.

File: CCPPtechnical/source/ParamSpecificOutput.rst (hunk 2 above)
Comment (grantfirl) [reply CCPPtechnical/source/ParamSpecificOutput.rst:187]: Yep, this is fine.

**Pull request merged**